http/director/router/adhoc.go

```diff
@@ -0,0 +1,109 @@
+package router
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+	"strings"
+
+	pb "github.com/mwitkow/kedge/_protogen/kedge/config/http/routes"
+)
+
+var (
+	// DefaultALookup is the lookup resolver for DNS A records.
+	// You can override it for caching or testing.
+	DefaultALookup = net.LookupAddr
```

Comment:
Its a very easy fix though, also makes much easier to reason about what lookup is used at the construction time.